app/assets/javascripts/snowcrash/modules/notifications.js.coffee

```diff
@@ -0,0 +1,12 @@
+document.addEventListener 'turbolinks:load', ->
+  $dropdown = $('[data-id="js-notifications-dropdown"]')
```

Comment:
let's stop using "js-" CSS classes for JS behaviors. Please update to: `$('[data-behavior~=notifications-dropdown]')`